hack/tools/tilt-prepare/main.go

```diff
@@ -70,17 +70,22 @@ var (
 	rootPath      string
 	tiltBuildPath string
 
-	toolsFlag            = pflag.StringSlice("tools", []string{}, "list of tools to be created; each value should correspond to a make target")
-	certManagerFlag      = pflag.Bool("cert-manager", false, "prepare cert-manager")
-	kustomizeBuildsFlag  = pflag.StringSlice("kustomize-builds", []string{}, "list of kustomize build to be run; each value should be in the form name:path")
-	providersBuildsFlag  = pflag.StringSlice("providers", []string{}, "list of providers to be installed; each value should be in the form name:path")
-	allowK8SContextsFlag = pflag.StringSlice("allow-k8s-contexts", []string{}, "Specifies that Tilt is allowed to run against the specified k8s context name; Kind is automatically allowed")
+	toolsFlag            = pflag.CommandLine.MarkDeprecated("tools", "list of tools to be created; each value should correspond to a make target")
```

Comment:
I think for tilt-prepare we don't need to deprecate flags and should instead just remove

Reply:
Yes done